src/transport/SecureSessionMgr.h

```diff
@@ -123,6 +123,12 @@ class EncryptedPacketBufferHandle final : private System::PacketBufferHandle
 class DLL_EXPORT SecureSessionMgrDelegate
 {
 public:
+    enum class DuplicateMessage : uint8_t
+    {
+        kYes,
+        kNo,
```

Comment:
```suggestion
        Yes,
        No,
```
if we do keep this enum class.